politeiawww/cmd/pictl/print.go

```diff
@@ -52,3 +55,53 @@ func byteCountSI(b int64) string {
 func printInPlace(s string) {
 	fmt.Printf("\033[2K\r%s", s)
 }
+
+// dollars converts an int64 price in cents into a human readable dollar
+// string.
+//
+// | Input     | Output          |
+// |-----------------------------|
+// | 13000     | "$130.00"       |
+// | 130000    | "$1,300.00"     |
+// | 13000000  | "$130,000.00"   |
+// | 130000000 | "$1,300,000.00" |
+// | 78        | "$0.78"         |
+// | -78       | "-$0.78"        |
+func dollars(cents int64) string {
+	// Get the value in dollars
+	dollarsValue := float64(cents) / 100
+
+	// Initial the buffer and check the duality of the value
+	buf := &bytes.Buffer{}
+	if dollarsValue < 0 {
+		buf.Write([]byte{'-'})
+		dollarsValue = 0 - dollarsValue
```

Comment:
It's not obvious why this is being done.  The reason is because the code below this expects a positive value and will not work correctly if the value is negative.  I would add a comment that clarifies this.  Example:

```
		// Convert the negative value to a positive value. The code
		// below can only handle positive values.
```